debug-bar.php

```diff
@@ -8,11 +8,11 @@
  Author URI: https://wordpress.org/
  Text Domain: debug-bar
  */
-add_filter( 'debug_bar_enable', function( $enable ) {
-	$enable = is_automattician();
 
-	return $enable;
-}, 99 );
+// If the user is an Automattician (typically a vip_support user), then force-enable Debug Bar.
+add_filter( 'debug_bar_enable', function( $enable ) {
+	return is_automattician() ? true : $enable;
+}, 999 );
```

Comment:
I find PHP_INT_MAX is a better priority for filters we want to give highest preference to:

```suggestion
}, PHP_INT_MAX );
```